README.md

```diff
@@ -1,70 +1,49 @@
-IBM_DB_SA
+[![Build Status](https://travis-ci.com/IBM/sqlalchemy-ibmi.svg?branch=master)](https://travis-ci.com/IBM/sqlalchemy-ibmi)
+[![Latest version released on PyPi](https://img.shields.io/pypi/v/sqlalchemy-ibmi.svg)]()
+[![](https://img.shields.io/pypi/pyversions/sqlalchemy-ibmi.svg)]()
+[![Documentation Status](https://readthedocs.org/projects/sqlalchemy-ibmi/badge/?version=latest)]()
+
+SQLAlchemy-IBMi
 =========
 
-The IBM_DB_SA adapter provides the Python/SQLAlchemy interface to IBM Data Servers.
+The SQLAlchemy-IBMi adapter provides the Python/[SQLAlchemy](https://www.sqlalchemy.org/) interface to Db2 for [IBM i](https://en.wikipedia.org/wiki/IBM_i).
```

Comment:
```suggestion
The  IBM i SQLAlchemy adapter provides an [SQLAlchemy](https://www.sqlalchemy.org/) interface to Db2 for [IBM i](https://en.wikipedia.org/wiki/IBM_i).
```

Nice, I like the added links. I think we should refer to it as "the IBM i SQLAlchemy adapter" instead of camel-casing/titleizing the package name, though.

Reply:
Can you also mention here that this project is experiencing a lot of changes/cleanups and is not ready for use. Using `:rotating_light:` might also be good.